content/ADR-172-rentals.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: adr
+adr: 172 
+title: Rentals
+date: 2023-01-10
+status: Living
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - fzavalia
+---
+
+## Abstract
+
+This document will talk about the Rentals Smart Contract on the Ethereum network that enables the rental of Land (Parcels and Estates) and other compatible ERC721 assets.
+
+## Context, Reach & Prioritization
+
+Rentals are a great way to earn passive income over assets. This will benefit owners that currently hold Land but don't know or want to build on it, creators that might find it expensive to buy Land and deploy scenes on it, and the whole of Genesis City as more and more creators will have access to populate and give it more life.
+
+Being able to rent your own Land was something the community has been asking for a while now. Rental solutions do exist, from dapps and smart contracts deployed by the community, to rentals being handled off-chain in a peer-2-peer manner.
+
+However, unofficial protocols and off-chain solutions might not take in consideration the best practices for security to protect users as well as Decentraland's most precious assets. That is why Decentraland has developed their own protocol.
+
+### How it works?
+
+The [Rentals Smart Contract](https://etherscan.io/address/0x3a1469499d0be105d4f77045ca403a5f6dc2f3f5#code) allow users to accept Rental Listings and Offers. 
```

Comment:
I have a doubt here. Why are we linking the proxy contract instead of the implementation?

```suggestion
The [Rentals Smart Contract](https://etherscan.io/address/0x3a1469499d0be105d4f77045ca403a5f6dc2f3f5#code) allows users to accept Rental Listings and Offers. 
```